src/hb-ot-layout.cc

```diff
@@ -710,6 +719,8 @@ struct hb_collect_features_context_t
   private:
   hb_set_t visited_script;
   hb_set_t visited_langsys;
+  unsigned int langsys_count;
+  unsigned int script_count;
```

Comment:
Nit. Reorder the two so follows previous lines.

Reply:
Done.